static/src/javascripts-legacy/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -93,7 +93,8 @@ define([
     }
 
     function defaultCanEpicBeDisplayed(testConfig) {
-        var enoughTimeSinceLastContribution = daysSince(lastContributionDate) >= 180;
+        var enoughTimeSinceLastContribution =
+            testConfig.showToContributors ? true : daysSince(lastContributionDate) >= 180;
```

Comment:
Could this be ` testConfig.showToContributors || daysSince(lastContributionDate) >= 180;`